### ISSUE TITLE
PHP: people listing HTML markup needs improvement

### ISSUE THREAD

**Issue by knice** [ucsc/wcms-design-templates#136]
http://history-dev.ucsc.edu/faculty/profiles/

This HTML badly needs improvement. These issues will get flagged for invalid markup and accessibility issues.
1. block-level elements (`h3`) nested **inside** of inline elements (`a`)
2. multiple `id` attributes with the same name (`title`).

Ideally this content should be removed from the table-based HTML as well, but that isn't as pressing. I will try to style this stuff to match the current site, but this needs to be fixed ASAP after launch (or before if there is time).

![HTML in Faculty profile pages](https://dl.dropboxusercontent.com/u/229696/_share/screens/2014-07-31-141808.png)
